pure_pursuit:
  lookahead_distance: 1.0
  max_velocity: 2.0